sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -125,7 +129,7 @@ case class DescribeFunctionCommand(
 
           if (isExtended) {
             result :+
-              Row(s"Extended Usage:\n${replaceFunctionName(info.getExtended, info.getName)}")
+              Row(s"Extended Usage:${replaceFunctionName(info.getExtended, info.getName)}")
```

Comment:
@jodersky if you meant `stripMargin` here, I am able to do this but I didn't do because I thought it's not great when other guys fix the documentation in each function but if majority of you think it's better, I will definitely do that.